src/lib/components/identity/identity-details.svelte

```diff
@@ -32,7 +33,12 @@
 <div class="identity-details w-100">
     <div class="d-xl-flex align-items-center justify-content-between bg-light rounded p-4">
         <div class="d-flex align-items-center">
-            <Icon size={64} boxed boxColor={USER_ICONS[type]?.boxColor} type={USER_ICONS[type]?.icon} />
+            <Icon
```

Comment:
It is better to create a `type` default value in `Icon` component instead of modifying props in parent component

Reply:
i dont agree 🙏🏼